packages/foreman/rubygem-get_process_mem/rubygem-get_process_mem.spec

```diff
@@ -59,15 +65,16 @@ gem build %{gem_name}.gemspec
 
 %install
 mkdir -p %{buildroot}%{gem_dir}
-cp -pa .%{gem_dir}/* \
+cp -a .%{gem_dir}/* \
         %{buildroot}%{gem_dir}/
 
 %files
 %dir %{gem_instdir}
+%{gem_instdir}/.github
+%exclude %{gem_instdir}/.gitignore
+%exclude %{gem_instdir}/.travis.yml
+%{gem_instdir}/appveyor.yml
```

Comment:
This can still be excluded since it's CI config.

Reply:
To help ensure this systemically -- https://github.com/theforeman/foreman-packaging/pull/6414